frontend/server/src/Controllers/User.php

```diff
@@ -2206,7 +2225,10 @@ public static function apiRemoveExperiment(\OmegaUp\Request $r) {
         $r->ensureIdentity();
```

Comment:
```suggestion
         $r->ensureMainUserIdentity();
```

así no necesitas L2228.

Reply:
Listo